tests/mock_platform.h

```diff
@@ -55,6 +55,7 @@ class MockPlatform : public platform::Platform
     MOCK_METHOD(QString, default_driver, (), (const, override));
     MOCK_METHOD(QString, default_privileged_mounts, (), (const, override));
     MOCK_METHOD(bool, is_image_url_supported, (), (const, override));
+    MOCK_METHOD(QString, get_username, (), (const, override));
```

Comment:
I'm a little unsure why this is here as I don't see it used anywhere in this code.  Could you please explain?

Reply:
the `SmbMountHandler` implementation uses `MP_PLATFORM.get_username()` so it had to be mocked

Reply:
Ah, ok :+1: